### ISSUE TITLE
Swedbank Lietuva

### ISSUE THREAD

**Issue by deivpaukst** [PrivSec-dev/banking-apps-compat-report#244]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

Swedbank Lietuva

### Link to app

https://play.google.com/store/apps/details?id=lt.swedbank.mobile

### App version

21475

### Country of the app

Lithuania

### Build Number

TQ2A.230305.008.E1.2023032600

### Device list

Pixel 6a

### Profile app tested in

Owner profile

### Google Play installed

Not Installed

### Google Play services Network permission revoked?

- [ ] Revoked
- [ ] Not revoked
- [X] I did not have Google Play services installed

### Native code debugging

- [X] Enabled
- [ ] Disabled

### Exploit protection compatibility mode

- [ ] Enabled
- [X] Disabled

### Stock OS compatibility

- [X] Works
- [ ] Does not work
- [ ] Not tested

### Description of the app's functionality

App does work, however biometric unlock and paying by phone as card do not. You also can't use SmartID as a login method as it thinks the phone is "rooted" however could possibly be fixed with GPlay services. Mobile signature does work without GPlay services though.

**Comment by akc3n** [comment]
https://github.com/GrapheneOS/os-issue-tracker/issues/2635

**Comment by spring-onion** [comment]
@Mlsauron @deivpaukst For me it's crashing when native code debugging is **allowed**.

**Comment by deivpaukst** [comment]
Huh. Is it fine without it? I might try getting it again and seeing if I can log in with it disabled.

**Comment by spring-onion** [comment]
> Ignoring the app's native code debugging attempts just makes it work? That sure is interesting.

Yeah it's super strange, I've only seen this happen one other time.

**Comment by deivpaukst** [comment]
@spring-onion Hey, I had a thought recently. Should I edit my initial report to have updated information? It could be neat if someone opens it and sees up to date info immediately.

**Comment by odinis** [comment]
> > Okey, so I did test it and managed to log in and it no longer crashes. Ignoring the app's native code debugging attempts just makes it work? That sure is interesting.
> 
> so do the contactless payments work?

It does. Works with these settings on Pixel 10:
Exploit protection compatibility mode: disabled
Hardened memory allocator: disabled
Native code debugging: blocked
WebView JIT: enabled
Dynamic code loading via memory: allowed
Dynamic code loading via storage:  allowed
